metadata.rb

```diff
@@ -3,4 +3,9 @@
 license          "All rights reserved"
 description      "Installs/Configures qt"
 long_description IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version          "0.0.1"
```

Comment:
I was serious, but I guess it's adding a new recipe so it doesn't really matter. As dave said, this is public, so we don't know if anyone happens to be using it. It's nice to follow some semblance of semver in case someone is.